tests/ts_util.erl

```diff
@@ -315,6 +315,7 @@ get_valid_big_data(N) ->
         N + 0.1,
         get_bool(X),
         N + 100000,
+        get_varchar(),
```

Comment:
I added this in response to my schema change below. The 3 other tests I found using the big schema passed with the change.